### DISCUSSION TITLE
Obtenir un seul formulaire

### DISCUSSION THREAD

**Discussion by mbonitho** [MTESSDev/FRW#53] [Annonces]
Bonjour,

Est-ce qu'il y a un moyen de récupérer un seul formulaire an appelant l'API ? J'ai vu qu'on peut appeler la méthode ObtenirFormulairesIndividu en donnant un NoPublicFormulaire, mais la requête ne retourne aucun résultat (Je l'appelle une fois sans paramètres pour obtenir la liste complète, puis quand je l'appelle une autre fois avec un des NoPublicForm obtenus, l'appel retourne une liste vide).

Merci!

**Comment by vinnyboii** [comment]
Bonjour,

Normalement, il est possible d'appeler ObtenirFormulairesIndividu pour un seul formulaire.

Quels paramètres avez-vous fourni dans votre 2e appel précisément? Je l'ai testé de mon côté et ce fut concluant.

Désolé pour le délai de réponse.

**Comment by vinnyboii** [comment]
@mbonitho 
désolé mon erreur, nous avons constaté qu'il y a effectivement un problème avec le service.
je vais ouvrir un issue à cet effet.

Bonne journée

**Comment by anisite** [comment]
Sera traité par #80
